src/main/java/org/gridsuite/network/map/dto/mapper/BranchInfosMapper.java

```diff
@@ -39,14 +40,19 @@ public static ElementInfos toData(@NonNull final Identifiable<?> identifiable,
         final Branch<?> branch = (Branch<?>) identifiable;
         final Double dcPowerFactor = Optional.ofNullable(infoTypeParameters.getOptionalParameters().get(QUERY_PARAM_DC_POWERFACTOR))
                 .map(Double::valueOf).orElse(null);
+        final boolean loadOperationalLimitGroups = Optional.ofNullable(infoTypeParameters.getOptionalParameters().get(QUERY_PARAM_LOAD_OPERATIONAL_LIMIT_GROUPS))
+            .map(Boolean::valueOf).orElse(false);
```

Comment:
to be compatible with actual result
```suggestion
            .map(Boolean::valueOf).orElse(true);
```

Reply:
I think it's better to change default behavior for performances sake